### ISSUE TITLE
Poor positioning of sections in user guide

### ISSUE THREAD

**Issue by jeromeongithub** [jeromeongithub/pe#15]
![image.png](https://raw.githubusercontent.com/jeromeongithub/pe/main/files/93ea1cd3-8f24-4ef5-81e1-0c223f7a7de4.png)

In the first image, there is the start command and how it works. Naturally, a user reading the user guide would want to know what commands are available after doing the start command and going into workout mode. However, the next 3 sections are "achievements", "clear_achievements" and "ippt ..." and only after those 3 will users get to read what can be done after using the start command. The 3 sections mentioned are all unable to be used while in workout mode (after running start command). 
After looking at the user guide's table of contents again, I noticed that I can be redirected to the "Within your workout session" section. However, I still feel that a normal user would not want to scroll all the way back up or go to the table of contents just to figure out what to do next.
Perhaps start could be the last command in the "General" section so that the "Within your workout session" section comes up immediately after?

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># UG not well-formatted
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>After the start command in the UG, it proceeds to explain achievements, clear_achievements and ippt before explaining the commands within the workout session. Given that the 3 commands listed above cannot be called before the existing exercise is completed, it can cause confusion for readers who are unable to run the 3 commands.
>
>
>
>
>
><hr>
><sub>[original: nus-cs2113-AY2223S2/pe-interim#449] [original labels: severity.Low type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>The reason for putting the achievements and clear_achievements right after the start command is because understanding how those functions work is necessary in order to fully understand the output shown upon the completion of an existing exercise.
>Hence we hope to be able to push this to severity very low.

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Medium`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
